tiphub-io/tiphub
# User Connections Tab

Closes #12, closes #11.

### What This Does

Adds a tab to profile that lets you add and remove connections to your existing profile. If you remove all of your connections, your account will be deleted.

### Steps to Test

1. Create an account
2. Add an additional connection via the connections tab, confirm it was added correctly
3. Remove that connection, confirm you are refreshed, and that going back to the tab shows it removed
4. Remove your last connection, confirm you were warned, and that your account was deleted after removal

## Screenshots

<img width="918" alt="Screen Shot 2019-05-07 at 11 53 04 AM" src="https://user-images.githubusercontent.com/649992/57314164-1b43bf80-70bf-11e9-8b82-3ca07c21464b.png">

<img width="585" alt="Screen Shot 2019-05-07 at 11 53 08 AM" src="https://user-images.githubusercontent.com/649992/57314169-1da61980-70bf-11e9-9386-3b36314d2e45.png">

<img width="571" alt="Screen Shot 2019-05-07 at 11 53 21 AM" src="https://user-images.githubusercontent.com/649992/57314171-1f6fdd00-70bf-11e9-834d-79797252354a.png">
